http-server/apache2-web-server/README.md

```diff
@@ -24,48 +28,63 @@ $ cd ~/httpd/
 Then, pull the container from the library:
 
 ```
-$ singularity pull library://sylabs/examples/httpd.sif:latest
+$ singularity pull library://sylabs/examples/httpd:latest
 ```
-If your running on `32 bit` os, you will need to build from a definition file, click [here](#building-the-contianer-from-a-definition-file) or scroll down for instructions.
+If you're running on `32 bit` os, you will need to build from a definition file, click [here](#building-the-contianer-from-a-definition-file) or scroll down for instructions.
```

Comment:
Maybe this hyperlink needs to be updated.. or ref needs to be added? takes me to the same section still